modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro

```diff
@@ -677,6 +677,47 @@ definition {
 		return ${managerId};
 	}
 
+	macro createObjectDefinitionWithAllKindsOfFieldTypes {
+		var objectDefinitionId = ObjectDefinitionAPI.createAndPublishObjectDefinition(
+			en_US_label = ${en_US_label},
+			en_US_plural_label = ${en_US_plural_label},
+			name = ${objectDefinitionName},
+			requiredStringFieldName = ${requiredStringFieldName});
+
+		if (isSet(picklistId)) {
+			ObjectDefinitionAPI.postObjectDefinitionObjectField(
+				businessType = "Picklist",
+				dBType = "String",
+				listTypeDefinitionId = ${picklistId},
```

Comment:
When taking a look at the codes, it seem we can reduce one paremater, 
if (isSet(picklistName)) {
 var picklistId = JSONObject.getPicklistId(picklistName = "customPicklist");
ObjectDefinitionAPI.postObjectDefinitionObjectField(
...
)}